keyboards/keychron/k4/rgb/v2/via/config.h

```diff
@@ -0,0 +1,7 @@
+
+#pragma once
+
+#define WEAR_LEVELING_SN32_EMULATION_PAGE_COUNT 32
+#define LAYER_STATE_8BIT
+#define DYNAMIC_KEYMAP_LAYER_COUNT 2
+#define DYNAMIC_KEYMAP_EEPROM_MAX_ADDR 1023
```

Comment:
with the wear leveling eeprom driver, there should be no reason to mess with addresses
also, 4 dynamic layers are probably possible, by bumping `WEAR_LEVELING_SN32_EMULATION_PAGE_COUNT`